pep/app/pods/components/modal-dialogs/help/preferences/template.hbs

```diff
@@ -48,6 +48,9 @@
         </div>
     </Modal.body>
     <Modal.footer>
-        <Button @type="primary" @label={{t "common.done"}} @wide={{true}} @action={{this.done}} />
+        {{#if this.updatePreferenceTask.isRunning}}
+            <small class="text-muted font-weight-bold mr-auto">{{t "common.saving"}}</small>
+        {{/if}}
+        <Button @type="primary" @label={{t "common.done"}} @wide={{true}} @action={{this.done}} @disabled={{this.updatePreferenceTask.isRunning}} />
```

Comment:
i think you also need to set `@closeable` (or w/e the arg is) to false on the `<ModalDialog>` itself while its running, so that they can't close it via Esc or the "x" header button

Reply:
@billdami doesn't look like there's a `@closeable` arg (or anything of the sort) 😢

Reply:
@napafundi this one I think? https://github.com/Gavant/gavant-ember-modals/blob/master/addon/components/modal-dialog.ts#L47

Reply:
@billdami d'oh... I searched `close` in those files 🤦